client/my-sites/site-settings/site-admin-interface/index.js

```diff
@@ -122,9 +122,7 @@ const SiteAdminInterface = ( { siteId, siteSlug, isHosting } ) => {
 						/>
 					</FormLabel>
 					<FormSettingExplanation>
-						{ hasEnTranslation( 'Use WP-Admin to manage your site.' )
-							? translate( 'Use WP-Admin to manage your site.' )
-							: translate( 'The classic WP-Admin WordPress interface.' ) }
+						{ translate( 'Use WP-Admin to manage your site.' ) }
```

Comment:
Removed the translation check because it's fully translated now (see https://translate.wordpress.com/deliverables/overview/11265049/).